### PR TITLE
Katies/vimeo api

### PR DESCRIPTION
**Pull request by ktrops**

### CODE REVIEW

File: app/controllers/instagram_controller.rb [hunk 1]
```diff
@@ -6,8 +6,11 @@ class InstagramController < ApplicationController
   FEED_URI_B = "/media/recent?client_id=#{ ENV["INSTAGRAM_CLIENT_ID"] }"
 
   def search
```
Comment (sojeri) [inline app/controllers/instagram_controller.rb]: We're not using this method anymore, right? Can you delete it? :)

File: app/controllers/vimeo_controller.rb [hunk 2]
```diff
@@ -1,3 +1,18 @@
...
-  # SEARCH_URI = {METHOD}https://api.vimeo.com
+   SEARCH_URI = "https://api.vimeo.com/users?per_page=30&query="
+   TOKEN_HEADER = {  
+                    "Accept" => "application/vnd.vimeo.*+json;version=3.2",
+                    "Authorization" => "bearer #{ENV["VIMEO_ACCESS_TOKEN"]}" 
+                  }
+   
+   
+  def results
+    user = params[:query]
```
Comment (sojeri) [inline app/controllers/vimeo_controller.rb]: can this variable be renamed to query or search_term? (something more descriptive of what the variable is doing?)

File: app/views/shared/_search_bar.html.erb [hunk 3]
```diff
@@ -1,4 +1,6 @@
-<%= form_for :search, url: search_path do |f| %>
+
+<%= form_for :search, url: search_path, method: :post do |f| %>
+
```
Comment (Brennaleker) [inline app/views/shared/_search_bar.html.erb:2]: we shouldn't need the method: :post anymore, it should function without it :)

File: app/views/vimeo/results.html.erb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+
+
+<h1>results</h1>
+<% @results.each do |result| %>
+  <%= result["name"] %>
+  <%= result["link"] %>
+<% end %>
```
Comment (vikshab) [inline app/views/vimeo/results.html.erb:5]: add a new line :)

File: app/controllers/vimeo_controller.rb [hunk 5]
```diff
@@ -1,3 +1,18 @@
...
+                
+    auth = JSON.parse(HTTParty.get(SEARCH_URI + user + "&sort=relevant", :headers => TOKEN_HEADER ))
+    @results = auth["data"]
+  end
+
+
+
```
Comment (vikshab) [inline app/controllers/vimeo_controller.rb]: Extra empty lines between ends

File: app/controllers/vimeo_controller.rb [hunk 6]
```diff
@@ -1,3 +1,18 @@
 class VimeoController < ApplicationController
-  # SEARCH_URI = {METHOD}https://api.vimeo.com
+   SEARCH_URI = "https://api.vimeo.com/users?per_page=30&query="
+   TOKEN_HEADER = {  
+                    "Accept" => "application/vnd.vimeo.*+json;version=3.2",
+                    "Authorization" => "bearer #{ENV["VIMEO_ACCESS_TOKEN"]}" 
+                  }
+
```
Comment (vikshab) [inline app/controllers/vimeo_controller.rb]: extra lines, plz delete one

File: app/views/instagram/individual_feed.html.erb [hunk 7]
```diff
@@ -1,3 +1,5 @@
+
```
Comment (vikshab) [inline app/views/instagram/individual_feed.html.erb]: extra empty line

File: app/views/vimeo/results.html.erb [hunk 8]
```diff
@@ -0,0 +1,7 @@
+
+
```
Comment (vikshab) [inline app/views/vimeo/results.html.erb]: Delete one extra empty line

**Comment by sojeri** [comment]
thanks for taking care of all our empty line issues! n_n

**Pull request merged**